packages/office-ui-fabric-react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -13,37 +12,59 @@ export class FabricBase extends React.Component<
 > {
   private _rootElement = React.createRef<HTMLDivElement>();
   private _disposables: (() => void)[] = [];
+  private _removeClassNameFromBody?: () => void = undefined;
 
   constructor(props: IFabricProps) {
     super(props);
     this.state = { isFocusVisible: false };
   }
 
   public render() {
-    const { className } = this.props;
-
-    const classNames = getClassNames(getStyles, {
-      theme: this.props.theme!,
-      className,
-      isFocusVisible: this.state.isFocusVisible
-    });
+    const classNames = this._getClassNamesHelper();
     const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties);
-
     return <div {...divProps} className={classNames.root} ref={this._rootElement} />;
   }
 
   public componentDidMount(): void {
     const win = getWindow(this._rootElement.current);
-
     if (win) {
       this._disposables.push(on(win, 'mousedown', this._onMouseDown, true), on(win, 'keydown', this._onKeyDown, true));
     }
+    this._addClassNameToBody();
   }
 
   public componentWillUnmount(): void {
     this._disposables.forEach((dispose: () => void) => dispose());
+    if (this._removeClassNameFromBody) {
+      this._removeClassNameFromBody();
+    }
   }
 
+  private _getClassNamesHelper(): IProcessedStyleSet<IFabricStyles> {
+    const { className, theme, applyTheme, applyThemeToBody } = this.props;
+    const classNames = getClassNames(getStyles, {
+      theme: theme!,
+      applyTheme: applyTheme,
+      applyThemeToBody: applyThemeToBody,
```

Comment:
I'm not seeing `applyThemeToBody` used in the style function.

Reply:
Oh yes `bodyThemed` which is the corresponding class in the styles file is added conditionally based on `this.props.applyThemeToBody` so we don't need it here in the getClassNames function